pilot/cmd/pilot-agent/status/server.go

```diff
@@ -95,6 +101,19 @@ type Server struct {
 	envoyStatsPort      int
 }
 
+func init() {
+	registry := prometheus.NewRegistry()
+	wrapped := prometheus.WrapRegistererWith(map[string]string{"agent": "istio"}, prometheus.Registerer(registry))
```

Comment:
```suggestion
    wrapped := prometheus.WrapRegistererWithPrefix("istio_agent", prometheus.Registerer(registry))
```

otherwise this will just add a label, but we need a prefix so it doesn't collide. We can probably at the go collector metrics then